merino/middleware/logging.py

```diff
@@ -26,6 +26,8 @@ async def dispatch(self, request: Request, call_next):
             "time": datetime.fromtimestamp(time.time()).isoformat(),
             # Provided by the asgi-correlation-id middleware.
             "rid": response.headers["X-Request-ID"],
+            "session_id": request.query_params.get("sid"),
+            "sequence_no": request.query_params.get("seq"),
```

Comment:
@tiftran By specs, `sequence_no` should be zero-based integers, let's do the conversion here.
```suggestion
            "sequence_no": int(seq) if (seq := request.query_params.get("seq")) else None
```

Reply:
oh snap you're right

Reply:
I think everything in `query_params` is a string (i could be wrong please verify), and "0" is a truthy value in python. so we should be good.

Reply:
Oooh 🤦‍♀️  you're right it's "0" not 0